### ISSUE TITLE
Copy wiki over

### ISSUE THREAD

**Issue by mrfishie** [MarkTime/MarkTimeJSv2#1]
We need to copy the wiki from [the old project](https://github.com/MarkTime/MarkTimeJS), as well as update the information for this new version.

**Comment by somebody1234** [comment]
@mrfishie Can you add a custom footer in the wiki? (with `<sub>[[Home]] | [[About MarkTimeJS|About]] | [[[Setup Guide|Setup]] | [[Technical Documentation]] | Copyright © 2013-2014 [[The MarkTime Team|MarkTime Team]]</sub>`) (and a sidebar)

**Comment by mrfishie** [comment]
Yes, you can, but what does that have to do with copying the wiki?

**Comment by somebody1234** [comment]
1. I can't do it
2. Doesn't that count as part of the wiki? It should be the only thing left to copy.

**Comment by mrfishie** [comment]
Ah, did you mean to say "can you add a custom footer"? (as opposed to "can't you add a custom footer")
Just noticed you added a bunch of pages on the wiki.. Thanks! Most of the pages will need to be updated later on, because many things have changed in this version. As a result I don't think the sidebar should be added just yet as lots of things will change.
I've added the footer though. If you can't see the 'add a footer' or 'add a sidebar' buttons on the wiki pages, you can simply create/edit the '_footer' and '_sidebar' pages.

**Comment by Baralabite** [comment]
Excuse my ignorance - and no offence intended, but who are you somebody1234?

**Comment by somebody1234** [comment]
Well, mrfishie is in my friend's organisation, and I was bored yesterday.
